src/posix/platform/netif.cpp

```diff
@@ -321,22 +666,89 @@ static void processNetifAddrEvent(otInstance *aInstance, struct nlmsghdr *aNetli
 
                     netAddr.mAddress      = addr;
                     netAddr.mPrefixLength = ifaddr->ifa_prefixlen;
-                    SuccessOrExit(error = otIp6AddUnicastAddress(aInstance, &netAddr));
+
+                    error = otIp6AddUnicastAddress(aInstance, &netAddr);
+
+                    if ((error == OT_ERROR_NONE) || (error == OT_ERROR_ALREADY))
+                    {
+                        otLogNotePlat(
```

Comment:
These logging code seems duplicate. Can we add a macro or function for them?

Reply:
I consolidated a lot of the logging that was making this so complicated.  Take a peek.